src/actions.ts

```diff
@@ -1,12 +1,13 @@
 import { RESET, UPDATE_SELECTED_STOCK } from "./store/constants";
+import { ISelectedSearch } from "./features/headline/redux/actions";
```

Comment:
nit: are you still using interfaces? the I prefix naming convention is typically for interfaces only

Reply:
Yeah I am, will change to `types` when refactoring later